lua/astrocommunity/pack/haskell/haskell.lua

```diff
@@ -37,20 +32,12 @@ return {
       -- TODO: Need haskell to be pulled into mason-nvim-dap (haskell hopefully coming soon)
       -- {
       --   "jay-babu/mason-nvim-dap.nvim",
-      --   opts = function(_, opts)
-      --     -- Ensure that opts.ensure_installed exists and is a table.
-      --     if not opts.ensure_installed then opts.ensure_installed = {} end
-      --     utils.list_insert_unique(opts.ensure_installed, "haskell-dap")
-      --   end,
+      --   opts = function(_, opts) opts.ensure_installed = utils.list_insert_unique(opts.ensure_installed, "haskell") end,
       -- },
```

Comment:
[The PR has already been merged](https://github.com/jay-babu/mason-nvim-dap.nvim/commit/7c18898e3a8ed369bfa0cfcc4be7bccdf5d13ea7)